spec/models/wishlist_spec.rb

```diff
@@ -3,4 +3,10 @@
 RSpec.describe Wishlist, type: :model do
   it { is_expected.to belong_to :user }
   it { is_expected.to have_many :items }
+
+  it "creates a token for every new wishlist created" do
+    # new_wishlist could be changed for let: when more tests are present
+    new_wishlist = Wishlist.create(name: "Name", description: "Description")
```

Comment:
As I remember smejfu have told me to use let also for variables which appears only once.